alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -385,6 +392,14 @@ class DocumentLevelMonitorRunner : MonitorRunner() {
                 RestStatus.INTERNAL_SERVER_ERROR,
                 e
             )
+            if (monitor.dataSources.queryIndex.contains("optimized") &&
```

Comment:
plz dont write this twice
move this to finally block

Reply:
fixed it.